### PR TITLE
Fixes for argument/variables analysis

### PR DESCRIPTION
**Pull request by GustavoLCR**
Before:

image

After:

image

----
Also, is there a way to merge 2 variables? For example in the second image var_18h and var_18h_3 are the same

### CODE REVIEW

**Comment by radare** [comment]
no, there's no command or api to merge local variables, it will be good to have it

**Comment by radare** [comment]
can you update the tests after rebasing?

File: test/new/db/cmd/types [hunk 1]
```diff
@@ -1259,7 +1259,7 @@ af
 aaft
 afv
 EXPECT=<<RUN
-var int32_t var_0h @ sp+0x0
+var int32_t var_1454h @ sp+0x0
```
Comment (radare) [inline test/new/db/cmd/types]: uhm is this correct ?

Comment (GustavoLCR) [reply test/new/db/cmd/types]: Stack grows to that size:

image

**Pull request merged**